core/src/test/java/org/apache/calcite/rel/rules/DateRangeRulesTest.java

```diff
@@ -329,6 +329,10 @@ private void checkDateRange(Fixture f, RexNode e, Matcher<String> matcher,
     private final RexNode exMonth;
     private final RexNode exDay;
 
+    private final RexNode exYearDt;
+    private final RexNode exMonthDt;
+    private final RexNode exDayDt;
+
```

Comment:
I'm going to rename `exYearDt` to `exYearD` because `Dt` looks misleading like 'dateTime`.